### PR TITLE
Password reset

### PR DESCRIPTION
**Pull request by noctera**
| Status  | Type  |
| :---: | :---: |
| :white_check_mark: Ready | Feature |

## Description

Added confirm dialog to change password

## Screenshots / GIFs (if appropriate):

Unbenannt

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

File: src/screens/Profile/Profile.jsx [hunk 1]
```diff
@@ -124,23 +161,54 @@ const Profile = () => {
           </div>
         </div>
       </div>
-      <Modal
-        title={t("screens.profile.modals.changePassword.heading")}
-        open={showPasswordModal}
+
+      <ConfirmDialog
```
Comment (luwol03) [inline src/screens/Profile/Profile.jsx:165]: add a submit label property.
image

**Review by luwol03 — CHANGES_REQUESTED** [review]

File: src/screens/Profile/Profile.jsx (hunk 1 above)
Comment (noctera) [reply src/screens/Profile/Profile.jsx:165]: You mean showSnack?

Comment (zikowang) [reply src/screens/Profile/Profile.jsx:165]: The button says "Delete" instead of "Submit".

Comment (noctera) [reply src/screens/Profile/Profile.jsx:165]: Oh, I didn't even recognize this

File: src/screens/Profile/Profile.jsx [hunk 2]
```diff
@@ -124,23 +161,55 @@ const Profile = () => {
...
+        <TextInput
+          required
+          placeholder={t("modal.changePassword.oldPassword")}
+          onChange={(value) => {
+            setOldPassword(value);
+          }}
+          value={oldPassword}
+          error={oldPasswordError}
+          errorText={t("modal.changePassword.oldPasswordError")}
+        />
```
Comment (luwol03) [inline src/screens/Profile/Profile.jsx:183]:
```suggestion
          type="password"
        />
```

Comment (noctera) [reply src/screens/Profile/Profile.jsx:184]: done

**Review by zikowang — APPROVED** [review]

**Review by luwol03 — APPROVED** [review]

**Pull request merged**